### PR TITLE
[charts/portal] Addressing the broken doc links in portal home page and in Api Explorer page

### PR DESCRIPTION
**Pull request by premchandervodela**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: charts/portal/values-production.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ global:
   legacyHostnames: false
   legacyDatabaseNames: false
   subdomainPrefix: dev-portal
-  helpPage: https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/
+  helpPage: https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-1/
```
Comment (ksaladi) [inline charts/portal/values-production.yaml:23]: I tried with link '5-0-2' not working, so assuming 5-1 also will not work. So lets keep till 'https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal'

Comment (premchandervodela) [reply charts/portal/values-production.yaml:23]: 5-1 is not yet published , I tested with 5-1test and it worked for me . If we keep till api-developer-portal - the links inside the portal - like landing page/home page and in api explorer page will not work.

Comment (ksaladi) [reply charts/portal/values-production.yaml:23]: we can change when 5-1 is available, for now lets remove.

**Review by ksaladi — APPROVED** [review]
LGTM

**Pull request merged**